domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/RevurderingRepo.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.su.se.bakover.domain.revurdering
+
+import no.nav.su.se.bakover.domain.behandling.Attestering
+import java.util.UUID
+
+interface RevurderingRepo {
+    fun hent(id: UUID): AbstraktRevurdering?
+    fun hentEventuellTidligereAttestering(id: UUID): Attestering?
```

Comment:
Brukes denne lenger når vi har den attesteringshistorikken?

Reply:
Ikke relevant for denne PR'en kanskje, men ja. Den brukes i `RevurderingServiceImpl.sendTilAttestering()`. Enig i at det fremstår som overflødig hvis det eksisterer en attesteringshistorikk.